README.md

```diff
@@ -23,17 +23,19 @@ There are two launch files included as samples in the launch folder. `tracker.la
 
 To use hardware accelleration, install [CUDA Toolkit 10.1](https://developer.nvidia.com/cuda-10.1-download-archive-base) and [cuDNN v7 for CUDA 10.1](https://developer.nvidia.com/rdp/cudnn-archive). 
 
-For your own project, you can create a launch file in the following format:
+For a project trained using customvision.ai, you can replace the tracker_type to "customvision", place the relevant extracted onnx zip folder to the `ros_msft_onnx/testdata/` folder and edit the launch file in the following format:
```

Comment:
Does it come with some documentation about the pre-processing and post-processing sample code or doc? It would be great to have a reference link added somewhere in the README so we can cross reference in future.

Reply:
the onnx model from customvision creates a mode l like this(see below). 
![model](https://user-images.githubusercontent.com/25962122/110399292-741d6f80-802a-11eb-9078-08c8d0b5ec87.PNG)

Reply:
@seanyen could not find a sample code/doc but from my investigation the input is similar to that of the yolo mode 1x3x416x416 with the cell width and height of 32x32 and  the image is resized based on this info(fun fact the onnx model output changes to none x none x different_number x different_number when the input size is changed from the default 416 x 416 x 3). The output channels are 25 for one class. 30 for two classes and so on ..